File: aips assign18/task4.js
/**
 * Task 4: Data Structures with Functions (JavaScript → Python)
 * Function `printStudents(students)` prints each student name from an array.
 */

function printStudents(students) {
    console.log('Student List:');
    for (let i = 0; i < students.length; i++) {
        console.log(students[i]);
    }
}

// Test the function with sample student names
const students = ['Alice', 'Bob', 'Charlie'];
printStudents(students);

File: aips assign18/task4.py
"""
Task 4: Data Structures with Functions (JavaScript → Python)
This module contains a Python function that prints student names from a list.
"""

def print_students(students):
    """Print each student name from the provided list."""
    print("Student List:")
    for name in students:
        print(name)


if __name__ == "__main__":
    students = ['Alice', 'Bob', 'Charlie']
    print_students(students)
